tests/end-to-end/external-solution-integration/pkg/testsuite/check_event_id.go

```diff
@@ -0,0 +1,47 @@
+package testsuite
+
+import (
+	retrygo "github.com/avast/retry-go"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/retry"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/step"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/testkit"
+)
+
+// CheckEventId is a step which checks if the correct EventId has been received
+type CheckEventId struct {
```

Comment:
I could not find where is this used?

Reply:
Yes, this is a remainder from before the steps where combined and should have been removed! Fixed it now